<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NadminN</class>
 <widget class="QDialog" name="NadminN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="username_editBalance_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newBalance_editBalance_lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>New Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBalance_pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Balance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_Activate_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Activate_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_suspend_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suspend_pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Suspend</string>
   </property>
  </widget>
  <zorder>username_editBalance_lineEdit</zorder>
  <zorder>newBalance_editBalance_lineEdit</zorder>
  <zorder>editBalance_pushButton</zorder>
  <zorder>username_Activate_lineEdit</zorder>
  <zorder>username_suspend_lineEdit</zorder>
  <zorder>Activate_pushButton</zorder>
  <zorder>suspend_pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
